websites/P/ProBot/presence.ts

```diff
@@ -9,34 +9,165 @@ presence.on("UpdateData", () => {
 		startTimestamp: browsingTimestamp
 	};
 
-	if (document.location.pathname.includes("/dashboard")) {
-		presenceData.details = "Managing Server";
-		presenceData.state = (
-			document.querySelector("div.user-profile > div > div") as HTMLElement
-		).textContent;
+	if (document.location.pathname === "/")
+		presenceData.details = "Visiting the Main Page";
+	else if (document.location.pathname.includes("/dashboard")) {
+		presenceData.details = "Visiting the Dashboard";
+		presenceData.buttons = [
+			{
+				label: "Go to Dashboard",
+				url: document.URL
+			}
+		];
 	} else if (document.location.pathname.includes("/commands")) {
 		presenceData.details = "Viewing a page:";
 		presenceData.state = "Commands";
-	} else if (document.location.pathname.includes("/premium"))
-		presenceData.details = "Viewing Premium";
-	else if (document.location.pathname.includes("/bgs")) {
-		presenceData.details = "Managing the";
-		presenceData.state = "profile backgrounds";
-	} else if (document.location.pathname.includes("/id")) {
-		presenceData.details = "Managing the";
-		presenceData.state = "ID backgrounds";
+		presenceData.buttons = [
+			{
+				label: "View ProBot Commands",
+				url: document.URL
+			}
+		];
+	} else if (document.location.pathname.includes("premium")) {
+		if (document.location.pathname.includes("/manage"))
+			presenceData.details = "Managing their Premium";
```

Comment:
what does managing a premium mean? Also Premium should start with a lower case letter (same for all instances below)

Reply:
@ImRodry Premium goes for the servers (tier 1) and custom bots (tier 2). People can manage their premium server/bot settings there

Reply:
well then you need to say that. You don't manage the premium, if anything you manage the premium settings

Reply:
I've changed it to "Managing their Premium Bots/Servers", is that okay?

Reply:
You should avoid using slashes, I’d go with premium settings